website/src/specs/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "specs",
```

```suggestion
  "name": "@instantsearch/specs-website",
```

not important, as this package isn't installed, but best to choose names that don't mirror public packages maybe?